omod/src/main/resources/apps/edtriage_app.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+        "id": "edtriageapp.app.edTriage",
+        "instanceOf": "coreapps.template.findPatient",
+        "description": "Search for a study participant and take user to the Edit Contacts page",
+        "order": 20,
+        "config": {
+            "label": "ED Triage",
+            "afterSelectedUrl":"/edtriageapp/edtriageEditPatient.page?patientId={{ patientId }}",
```

Comment:
@pmanko @mozzy11 so on spinning this app within the ed triage module when isanteplus module is started,it never redirects to this page but on stopping the isanteplus module it does redirect